pkgs/edge-worker/src/EdgeWorker.ts

```
@@ -95,12 +97,29 @@ export class EdgeWorker {
+    };
 
-      // Ensure connectionString is always set
-      connectionString,
+    // Create a worker factory function that will be called by the adapter
+    const createWorkerFn: CreateWorkerFn = (createLoggerFn: CreateLoggerFn) => {
+      return createQueueWorker(handler, completeConfig, createLoggerFn);
     };
 
-    this.setupRequestHandler(handler, completeConfig);
+    // Initialize the adapter with the worker factory function
+    await this.adapter.initialize(createWorkerFn);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)